### PR TITLE
update annotation processing

### PR DESCRIPTION
**Pull request by birdage**
DO NOT MERGE

in progress

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher this needs to get merged in first before the UI portion can be merged

**Comment by DanielJMaher** [comment]
All checks have failed.  Mind fixing? @birdage

**Comment by birdage** [comment]
@DanielJMaher its to do with test coverage, will look at it

**Comment by DanielJMaher** [comment]
@FBRTMaka Check this please!!!!

**Comment by DanielJMaher** [comment]
@birdage What happened now?

**Comment by DanielJMaher** [comment]
======================================================================
ERROR: tests.test_annotations (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.test_annotations
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 252, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 230, in _get_module_from_name
    __import__(name)
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/tests/test_annotations.py", line 23, in <module>
    @skipIf(os.getenv('TRAVIS'), 'Skip if testing from Travis CI.')
NameError: name 'skipIf' is not defined

File: ooiservices/app/main/annotation.py [hunk 1]
```diff
@@ -15,103 +15,104 @@
...
-
+from ooiservices.app.uframe.controller import split_stream_name
 import json
+import requests
+
+#List all annotations. build 6
+@api.route('/annotation/<string:instrument>/<string:stream>')
+def get_annotations(instrument,stream):
+    try:
+        if 'startdate' in request.args and 'enddate' in request.args:
```
Comment (FBRTMaka) [inline ooiservices/app/main/annotation.py]: @birdage will this still fire if startdate and end date are empty?

**Comment by DanielJMaher** [comment]
======================================================================
FAIL: test_annotation (tests.test_annotations.AnnotationsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/tests/test_annotations.py", line 75, in test_annotation
    self.assertEquals(response.status_code, 201)
AssertionError: 500 != 201
======================================================================
FAIL: test_searching (tests.test_annotations.AnnotationsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/tests/test_annotations.py", line 167, in test_searching
    self.assertEquals(response.status_code, 201)
AssertionError: 500 != 201

File: ooiservices/app/main/annotation.py (hunk 1 above)
Comment (birdage) [inline ooiservices/app/main/annotation.py]: @FBRTMaka you will always need a start and end time. im going to change it to the other endpoint for annotations eventually, but we wanted to get it in.

**Comment by DanielJMaher** [comment]
@birdage PASSED

**Comment by DanielJMaher** [comment]
@FBRTMaka You cool with this?

File: ooiservices/app/main/annotation.py (hunk 1 above)
Comment (FBRTMaka) [inline ooiservices/app/main/annotation.py]: @birdage word, just bringing it to light!

**Comment by FBRTMaka** [comment]
@birdage totes

**Pull request merged**